### PR TITLE
fix(settings): Implement text block for blocks of text

### PR DESCRIPTION
**Pull request by ckj**
K, text in settings is a bit better now:

screen shot 2018-01-09 at 3 36 39 pm

I also went ahead and cleaned up the Audit log header and reverted the `flex-end` issue that was improperly aligning the switch component.

### CODE REVIEW

**Review by billyvg — APPROVED** [review]

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx [hunk 1]
```diff
@@ -28,7 +28,6 @@ const FormFieldControlErrorWrapper = styled(Box)`
 const FormFieldControlStyled = styled(FormFieldControl)`
   display: flex;
   flex-direction: column;
-  align-items: flex-end;
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx]: 🤔

File: src/sentry/static/sentry/app/views/settings/components/text/textBlock.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import styled from 'react-emotion';
+
+const TextBlock = styled.div`
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/text/textBlock.jsx]: why not a styled `<p/>`?

File: src/sentry/static/sentry/app/views/settings/organization/auditLog/auditLogList.jsx [hunk 3]
```diff
@@ -65,33 +64,27 @@ class AuditLogList extends React.Component {
     let {pageLinks, entries, eventType, eventTypes, onEventSelect} = this.props;
     let hasEntries = entries && entries.length > 0;
 
+    let action = (
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/organization/auditLog/auditLogList.jsx]: this seems unrelated...just confirming you meant to commit this?

File: src/sentry/static/sentry/app/views/settings/organization/rateLimit/rateLimitView.jsx [hunk 4]
```diff
@@ -56,6 +57,10 @@ class AccountLimit extends React.Component {
   }
 }
 
+const TextBlockStyled = styled(TextBlock)`
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/organization/rateLimit/rateLimitView.jsx]: this is a nice way of combining these styles, tho it still would be trouble if we ever wanted more than two (which maybe we don't so for now who cares)

File: src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx]: https://github.com/getsentry/sentry/pull/6915 introduced a regression (removing this also causes another one, specifically when form fields have `inline=true`).

Comment (Chrissy) [reply src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx]: 🤔 they both cause a regression?

Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx]: Yeah look at percy-storybook + 
https://percy.io/getsentry/sentry-storybook/builds/469815

File: src/sentry/static/sentry/app/views/settings/components/text/textBlock.jsx (hunk 2 above)
Comment (ckj) [reply src/sentry/static/sentry/app/views/settings/components/text/textBlock.jsx]: Since semantics matter less in this world, I was hoping to avoid the nested `<p>` problem you brought up when we discussed IRL by just styling a div.  I can style the `<p>` if you feel strongly about it.

File: src/sentry/static/sentry/app/views/settings/organization/auditLog/auditLogList.jsx (hunk 3 above)
Comment (ckj) [reply src/sentry/static/sentry/app/views/settings/organization/auditLog/auditLogList.jsx]: Yep! It is related in that there was an unnecessary blurb of text next to this control, so I removed it and made the control a page header action instead.

File: src/sentry/static/sentry/app/views/settings/organization/rateLimit/rateLimitView.jsx (hunk 4 above)
Comment (ckj) [reply src/sentry/static/sentry/app/views/settings/organization/rateLimit/rateLimitView.jsx]: Yeah, this was more of solving the one off issue of the component's default 30px margin being too much. The jury is out on whether or not this is better than simply adding the styles inline.

File: src/sentry/static/sentry/app/views/settings/components/text/textBlock.jsx (hunk 2 above)
Comment (Chrissy) [reply src/sentry/static/sentry/app/views/settings/components/text/textBlock.jsx]: That's a pretty convincing let's do it!
